src/components/index.ts

```diff
@@ -1,2 +1,26 @@
 export { Alert } from "./alert";
 export { InlineAlert } from "./inline-alert";
+export { Avatar } from "./avatar";
+export { Badge } from "./badge";
+export { Button } from "./button";
+export { Checkbox } from "./checkbox";
+export { Dialog } from "./dialog";
+export { DividerLine } from "./divider-line";
+export { FormField } from "./form-field";
+export { IconButton } from "./icon-button";
+export { LastChangedInfo } from "./last-changed-info";
+export { Menu } from "./menu";
+export { Page } from "./page";
+export { Section } from "./section";
+export { Sidebar } from "./sidebar";
+export { SidebarContainer } from "./sidebar-container";
+export { Sidesheet } from "./sidesheet";
+export { Spinner } from "./spinner";
+export { SpinnerOverlay } from "./spinner-overlay";
+export { Tab } from "./tab";
+export { TableUnvirtualized } from "./table-unvirtualized";
+export { TableVirtualized } from "./table-virtualized";
+export { Tag } from "./tag";
+export { Toast } from "./toast";
+export { Toggle } from "./toggle";
+export { TopBar } from "./top-bar";
```

Comment:
FYI, and not a stopper for this PR:
This highly depends on what's using this package, but I've seen this pattern create issues in the past. The problem is that when you're importing it, if you import any components from the root of the package, like `import { Tag } from "@abusix/hailstorm";`
it will make the compiler come to this index, and to give you the `Tag` component, it needs to resolve all the exports from this file (so it means traversing inside the folders), which makes type-checking and compiling noticeable slower.

We should be able to export the modules separately, so we would need to do something like `import { Tag } from "@abusix/hailstorm/tag";`, but it makes compiling and type resolution a lot faster on big projects.

We would use the "exports" field in the package.json for this, something like 

```json
"exports": {
  "./*": "./dist/components/*.js"
}
``` 
or 

```json
"exports": {
  "./": "./dist/components/"
}
``` 
But again, it depends on the compatibility of the libraries that are consuming this. The export field can have a big complicated config, depending on the cases we have to support.

Reply:
Good idea, I will create an issue to track it.